background/ip-addresses.rst

```diff
@@ -0,0 +1,69 @@
+.. _infrastructure-ip-addresses:
+
+IP adresses
+===============================
+
+Divio's containerised infrastructure is distributed over a number of geographical and vendor based regions. At any one
+time, the servers within each region are allocated an IP address from that region's range.
+
+Servers in our infrastucture (application builders and runners, and other services) are not permanent hosts but
+short-lived instances that can be provisioned at a moment's notice and are regularly recycled.
+
+The numbers in the pool of IP addresses within a region are therefore constantly being reallocated to different
```

Comment:
`numbers`? This should be addresses (but I guess you then want to reformulate to avoid repetition).